lib/common/xml.c

```diff
@@ -1291,18 +1304,27 @@ xml_patch_version_check(xmlNode *xml, xmlNode *patchset, int format)
 
     if(changed == FALSE) {
         crm_notice("Versions did not change in patch %d.%d.%d", add[0], add[1], add[2]);
-        return -pcmk_err_old_data;
+        return pcmk_rc_old_data;
     }
 
     crm_debug("Can apply patch %d.%d.%d to %d.%d.%d",
              add[0], add[1], add[2], this[0], this[1], this[2]);
-    return pcmk_ok;
+    return pcmk_rc_ok;
 }
 
+/*!
+ * \internal
+ * \brief Apply a version 1 patchset to an XML node
+ *
+ * \param[in,out] xml       XML to apply patchset to
+ * \param[in]     patchset  Patchset to apply
+ *
+ * \return Standard Pacemaker return code
+ */
 static int
 xml_apply_patchset_v1(xmlNode *xml, xmlNode *patchset)
```

Comment:
`apply_v1_patchset` would match with the other renamed functions better.

Reply:
updated